src/utils/utils.py

```diff
@@ -28,8 +28,8 @@ def get_arguments():
     
     # Important: prefix all heterogeneous GNNs names with "Hetero"
     parser.add_argument("gnn", choices=["GAT", "MessagePassing", "GraphSAGE", "GINE", "HeteroGNN"], default="GAT")
-    # if file is moved in another directory level relative to the root (currently in root/src), this needs to be changed
-    root_directory = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    # if file is moved in another directory level relative to the root (currently in root/utils/src), this needs to be changed
+    root_directory = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
I fixed this relative directories, otherwise with then new subfolder it would not locate the Data folder in root dir correctly.